src/appMain/sdl_preloaded_pt.json

```diff
@@ -1,6 +1,8 @@
 {
     "policy_table": {
         "module_config": {
+            "country_consent_passengersRC": true,

```

Comment:
@yang1070 
Some parts have been left to consider logic extension in future as this implementation is assumed as first part. Do you want to remove that?